controller/scheduler/fleet.py

```diff
@@ -187,6 +180,12 @@ def _create_container(self, name, image, command, unit, **kwargs):
         # encode command as utf-8
         if isinstance(l.get('command'), basestring):
             l['command'] = l['command'].encode('utf-8')
+
+        values = kwargs.get('values', {})
+        if values:
+            for key, value in values.items():
+                unit.append({"section": "Service", "name": "ExecStartPre", "value": '''/bin/sh -c "echo '{key}={value}' >> /tmp/env_files/{name}"'''.format(key=key, value=value)})
```

Comment:
Is it possible to put these files in `/etc/env_files/{name}` - which should be writable by the user creating these unit files